docs/daisy-migrating-existing-workflows.md

```diff
@@ -0,0 +1,110 @@
+
+Additionally, to avoid pinning your script to a specific release, you can use
+the image "family" as a partial URL, rather than naming a specific image.
+Partial URLs for image families have the following form:
+
+    projects/<project>/global/images/family/<family-name>
+
+You will need to find the correct project and family name for your chosen image
+family. To do this, navigate in the Cloud Console to Compute Engine -> Images,
+then click on the name of the instance you would like to use. On this page will
+be listed the image's family. To find the project, click on "Equivalent REST"
+and find the "selfLink" field. The second part of the path is the project name.
+Note that the project name is not the name of your own project, it is the name
+of the special project which contains your chosen base image. For example, the
+partial URL for the Ubuntu 16.04 LTS image family is:
+
+    projects/ubuntu-os-cloud/global/images/family/ubuntu-1604-lts
+
+You can use this partial URL in the "SourceImage" field in a "CreateDisks" Daisy
+step, or anywhere else that Daisy accepts partial URLs. By using an image
+family, you ensure that you will always run your workflow on the most up-to-date
+base image available.
+
+## Translating your Current Workflow
+
+We recommend that you write down every step you perform to prepare your image,
+in the order they are performed. This will be different for every image, but it
+usually involves some combination of downloading and running installers,
+building code, invoking package managers, and writing to configuration files.
+
+The biggest thing to keep in mind is that Daisy only allows you to run one
+script on startup. This script can download and run other scripts, but it must
+be the one to do it. Once you have listed all of the steps you take to set up
+your image, write a script that performs all of them.
+
+All of the files that your startup script needs to access (which are not already
+present on the machine) should be listed in the "Sources" field of your Daisy
+workflow. Sources can be paths to local files, or to files in a Storage bucket
+using the URL format `gs://<bucket>/<file>`.
+
+Files listed in the "Sources" field of your Daisy workflow are all copied to a
+special location in Google Cloud Storage. For more information on getting the
+URL of this location and downloading files from it, see "Passing Data to
+Instances."
+
+All Windows image creation startup scripts should end with a call to
+`gcesysprep.bat`. All other OS scripts should end by shutting down the instance.
+
+## A Typical Workflow
+
+The typical Daisy image creation workflow has five steps:
+
+    1. A CreateDisks step which creates at least one disk. The SourceImage for
+       one of these disks should be your base image.
+    2. A CreateInstances step which creates an instance out of the disks the
+       previous step created. The first disk in this step's "Disks" field must
```

Comment:
Here and everywhere else: I like to put field names in inline code blocks, e.g. \`Disks\` = `Disks`. Take it or leave it.

Reply:
Sounds good. Let's do that.